### ISSUE TITLE
AttributeError: term 'uri' not in namespace 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'

### ISSUE THREAD

**Issue by midannii** [usc-isi-i2/ontology-visualization#13]
I want to convert `uls_review_after.ttl` into  `after.dot`
I found out following error: 

```
$ python ./ontology_viz.py -o after.dot uls_review_after.ttl ontology.ttl
```

```
Traceback (most recent call last):
  File "./ontology_viz.py", line 23, in <module>
    common_ns = set(map(lambda ns: ns.uri, (RDF, RDFS, SKOS, SCHEMA, XSD, DOAP, FOAF)))
  File "./ontology_viz.py", line 23, in <lambda>
    common_ns = set(map(lambda ns: ns.uri, (RDF, RDFS, SKOS, SCHEMA, XSD, DOAP, FOAF)))
  File "/home/midannii/.conda/envs/midan2/lib/python3.7/site-packages/rdflib/namespace/__init__.py", line 212, in __getattr__
    return cls.__getitem__(name)
  File "/home/midannii/.conda/envs/midan2/lib/python3.7/site-packages/rdflib/namespace/__init__.py", line 202, in __getitem__
    raise AttributeError(f"term '{name}' not in namespace '{cls._NS}'")
AttributeError: term 'uri' not in namespace 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
```

**Comment by fatestigma** [comment]
I believe this is the same issue as #11 
And it has been addressed by #12 
I will close this issue now.

**Comment by dtraparic** [comment]
Hi, sorry to bother, I happened to get the exact same error, on Windows 10. With the following pip list: 
```
Package    Version
---------- -------
graphviz   0.20.1
isodate    0.6.1
pip        22.3
pyparsing  3.0.9
rdflib     6.2.0
setuptools 58.1.0
six        1.16.0
```

Running the example provided in readme.md:
```python .\ontology_viz.py -o test.dot test.ttl -O ontology.ttl```

Will trigger this error:
```python
Traceback (most recent call last):
  File "C:\Users\David Traparic\Documents\prog\ontology\ontology-visualization\ontology_viz.py", line 23, in <module>
    common_ns = set(map(lambda ns: ns.uri, (RDF, RDFS, SKOS, SCHEMA, XSD, DOAP, FOAF)))
  File "C:\Users\David Traparic\Documents\prog\ontology\ontology-visualization\ontology_viz.py", line 23, in <lambda>
    common_ns = set(map(lambda ns: ns.uri, (RDF, RDFS, SKOS, SCHEMA, XSD, DOAP, FOAF)))
  File "C:\Users\David Traparic\Documents\prog\ontology\ontoenv\lib\site-packages\rdflib\namespace\__init__.py", line 238, in __getattr__
    return cls.__getitem__(name)
  File "C:\Users\David Traparic\Documents\prog\ontology\ontoenv\lib\site-packages\rdflib\namespace\__init__.py", line 229, in __getitem__
    raise AttributeError(f"term '{name}' not in namespace '{cls._NS}'")
AttributeError: term 'uri' not in namespace 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
```

I tried different `rdflib` version, 6.1.1 will trigger the same error, whereas 6.0.1 and 5.0.0 will trigger another error:
```python
Traceback (most recent call last):
  File "C:\Users\David Traparic\Documents\prog\ontology\ontology-visualization\ontology_viz.py", line 6, in <module>
    from rdflib.plugins.sparql import prepareQuery
  File "C:\Users\David Traparic\Documents\prog\ontology\ontoenv\lib\site-packages\rdflib\plugins\sparql\__init__.py", line 33, in <module>
    from . import parser
  File "C:\Users\David Traparic\Documents\prog\ontology\ontoenv\lib\site-packages\rdflib\plugins\sparql\parser.py", line 184, in <module>
    Param('prefix', PN_PREFIX)) + Suppress(':').leaveWhitespace()
  File "C:\Users\David Traparic\Documents\prog\ontology\ontoenv\lib\site-packages\rdflib\plugins\sparql\parserutils.py", line 114, in __init__
    self.name = name
AttributeError: can't set attribute 'name'
```

 Do you have any ideas why this happens ?
